src/services/qrProcessor/index.js

```diff
@@ -12,8 +13,22 @@ export const decodeQrCode = qrCode => {
 export const encodeQrCode = payload =>
   `tradetrust://${encodeURIComponent(JSON.stringify(payload))}`;
 
+const decryptDocument = async uri => {
```

Comment:
think some error handling flows here would be good, and also the function name should get fetchAndDecryptDocument

1) what if decryption fails - could be unencrypted document?
2) what if key missing or uri malformed
3) what if fetch fails (timeout, 404, etc)